src/Config.php

```diff
@@ -409,7 +407,7 @@ private function setDataBuilder($config)
         }
         
         if (!isset($config['utilities'])) {
-            $config['utilities'] = $this->utilities;
+            $config['utilities'] = $this->utilities();
```

Comment:
This file is a good example of how the `utilities` call has been refactored into a more testable form with a trait + method call, instead of a property + injection.